ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-aro.yaml

```diff
@@ -0,0 +1,89 @@
+base_images:
+  ansible:
+    name: "4.11"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.11"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.11"
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.11"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.11"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.11"
+  tools:
+    name: "4.11"
+    namespace: ocp
+    tag: tools
+  upi-installer:
```

Comment:
I guess we also need "upi-installer:4.10" to keep it consistent with source cluster version.